locales/cy/translation.json

```diff
@@ -133,17 +133,14 @@
     },
     "pyiNoMatch": {
       "title": "Mae’n ddrwg gennym, ni allwn gadarnhau eich hunaniaeth",
-      "titleRepeatFraudCheck": "Mae'n ddrwg gennym, nid oeddem yn gallu cadarnhau eich manylion",
       "header": "Mae’n ddrwg gennym, ni allwn gadarnhau eich hunaniaeth",
-      "headerRepeatFraudCheck": "Mae'n ddrwg gennym, nid oeddem yn gallu cadarnhau eich manylion",
       "content": {
         "paragraph1": "Nid oeddem yn gallu dod o hyd i unrhyw un yn cyfateb pan wnaethom wirio’ch manylion gyda sefydliad arall.",
         "paragraph1Nino": "Nid oeddem yn gallu dod o hyd i wybodaeth sy’n cyfateb pan wnaethom wirio eich manylion gyda CThEF.",
         "paragraph1BankAccount": "Nid oeddem yn gallu dod o hyd i wybodaeth sy’n cyfateb pan wnaethom wirio eich manylion gyda’ch banc neu gymdeithas adeiladu.",
         "paragraph2": "Er mwyn cadw eich gwybodaeth yn ddiogel, ni allwn ddweud wrthych pa un o’ch manylion nad oeddem yn gallu cyfateb.",
         "subHeading": "Beth allwch chi ei wneud",
-        "paragraph3": "Parhewch i’r gwasanaeth roeddech yn ceisio ei ddefnyddio i ddarganfod a oes ffyrdd eraill o brofi eich hunaniaeth.",
-        "paragraph3RepeatFraudCheck": "Parhau i’r gwasanaeth roeddech yn ceisio ei ddefnyddio i ddarganfod a oes ffyrdd eraill o gael mynediad ato."
+        "paragraph3": "Os nad ydych yn siwr y gallwch profi eich hunaniaeth y ffordd hyn, gallwch fynd yn ôl i'r gwasanaeth roeddech yn ceisio ei ddefnyddio i ddarganfod os oes ffyrdd eraill i gael mynediad iddo."
```

Comment:
I assume you got the welsh translation from the appropriate place (can you link on the ticket, or is it just in slack?)

Reply:
have just linked on the ticket but it's I found it on the Welsh master spreadsheet: https://docs.google.com/spreadsheets/d/1b9G1o1jNjQLZXUxC5aFdDv4Hi0h_G_iimJWwliF9-ws/edit?gid=1863326233#gid=1863326233